doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,32 @@
-=======================
-Release Build Creation
-=======================
+===========================
+Compile & Build Cortx Stack
+===========================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
 
-   - You can use a Virtual Machine (VM) also.
+  - Please create VM(s) with at least 1 cores (vCPU) and 1GB of RAM.
+  - Ensure the VM is created with minimum of total 2 attached raw disks (additional disks apart from main bootable disk) per node
+
+    - Data Disks - Min 1 Disks per node (Capacity 20G+)
```

Comment:
This is incorrect. As per the spec, for 4+2+2 we would need 8 data disks + 1 metadata disks on each node. Description needs to be consistent. 

Ref: 
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/247595810/Motr+Scale+Out+Deployment+Architecture#Storage-Configuration.1

Reply:
done

Reply:
No, a community build of CORTX can have 1 drive.